packages/design-system/src/components/Table/TableHead.tsx

```diff
@@ -1,9 +1,19 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 
-export const TableHead = ({ children, ...tableHeadProps }) => {
+export interface TableHeadProps {
+  /**
+   * The table head contents, usually `TableRow`.
+   */
+  children?: React.ReactNode;
```

Comment:
Not sure if `_stackable?: boolean;` was needed here. I see it in the definition file.